src/main/java/com/fundation/logic/view/AdvancedSearchPanel.java

```diff
@@ -0,0 +1,158 @@
+        gbc_textFieldVideoBitrate.gridy = 2;
+        add(textFieldVideoBitrate, gbc_textFieldVideoBitrate);
+        textFieldVideoBitrate.setColumns(10);
+
+        JLabel lblFramerate = new JLabel("Framerate: ");
+        GridBagConstraints gbc_lblFramerate = new GridBagConstraints();
+        gbc_lblFramerate.anchor = GridBagConstraints.EAST;
+        gbc_lblFramerate.insets = new Insets(0, 0, 5, 5);
+        gbc_lblFramerate.gridx = 0;
+        gbc_lblFramerate.gridy = 3;
+        add(lblFramerate, gbc_lblFramerate);
+
+        textFieldVideoFramerate = new JTextField();
+        GridBagConstraints gbc_textFieldVideoFramerate = new GridBagConstraints();
+        gbc_textFieldVideoFramerate.insets = new Insets(0, 0, 5, 0);
+        gbc_textFieldVideoFramerate.fill = GridBagConstraints.HORIZONTAL;
+        gbc_textFieldVideoFramerate.gridx = 1;
+        gbc_textFieldVideoFramerate.gridy = 3;
+        add(textFieldVideoFramerate, gbc_textFieldVideoFramerate);
+        textFieldVideoFramerate.setColumns(10);
+
+        JLabel label = new JLabel("");
+        GridBagConstraints gbc_label = new GridBagConstraints();
+        gbc_label.insets = new Insets(0, 0, 5, 5);
+        gbc_label.gridx = 0;
+        gbc_label.gridy = 4;
+        add(label, gbc_label);
+
+        JSeparator separator = new JSeparator();
+        GridBagConstraints gbc_separator = new GridBagConstraints();
+        gbc_separator.insets = new Insets(0, 0, 5, 5);
+        gbc_separator.gridx = 0;
+        gbc_separator.gridy = 5;
+        add(separator, gbc_separator);
+
+        JLabel lblAudio = new JLabel("Audio");
+        lblAudio.setFont(new Font("Tahoma", Font.BOLD, 14));
+        GridBagConstraints gbc_lblAudio = new GridBagConstraints();
+        gbc_lblAudio.insets = new Insets(0, 0, 5, 0);
+        gbc_lblAudio.gridwidth = 2;
+        gbc_lblAudio.gridx = 0;
+        gbc_lblAudio.gridy = 6;
+        add(lblAudio, gbc_lblAudio);
+
+        JLabel lblCodecAudio = new JLabel("Codec: ");
+        GridBagConstraints gbc_lblCodecAudio = new GridBagConstraints();
+        gbc_lblCodecAudio.anchor = GridBagConstraints.EAST;
+        gbc_lblCodecAudio.insets = new Insets(0, 0, 5, 5);
+        gbc_lblCodecAudio.gridx = 0;
+        gbc_lblCodecAudio.gridy = 7;
+        add(lblCodecAudio, gbc_lblCodecAudio);
+
+        JComboBox comboBoxAudioCodec = new JComboBox();
+        GridBagConstraints gbc_comboBoxAudioCodec = new GridBagConstraints();
+        gbc_comboBoxAudioCodec.insets = new Insets(0, 0, 5, 0);
+        gbc_comboBoxAudioCodec.fill = GridBagConstraints.HORIZONTAL;
+        gbc_comboBoxAudioCodec.gridx = 1;
+        gbc_comboBoxAudioCodec.gridy = 7;
+        add(comboBoxAudioCodec, gbc_comboBoxAudioCodec);
+
+        JLabel lblChannels = new JLabel("Channels: ");
+        GridBagConstraints gbc_lblChannels = new GridBagConstraints();
+        gbc_lblChannels.anchor = GridBagConstraints.EAST;
+        gbc_lblChannels.insets = new Insets(0, 0, 5, 5);
```

Comment:
Magical numbers.

Reply:
Numbers of coordinates in GridBagLayout, please refactor in the next Sprint